### PR TITLE
Add IANA Considerations

### PR DESCRIPTION
**Pull request by igorlord**

### CODE REVIEW

**Review by ferrieux — COMMENTED** [review]
So, this equates "TP present, and empty" with value 0, both meaning "tolerate receiving", right ?
Also, since these are varints, value 0 may be represented as 0x00, 0x4000, 0x80000000 or 0xC000000000000000, right (though I guess those subnormal varints are discouraged) ?

**Comment by igorlord** [comment]
> So, this equates "TP present, and empty" with value 0, both meaning "tolerate receiving", right?

Not exactly.  "TP empty" is not valid, since TP requires an integer parameter. 

> Also, since these are varints, value 0 may be represented as 0x00, 0x4000, 0x80000000 or 0xC000000000000000, right (though I guess those subnormal varints are discouraged) ?

Yes.  That's how varints work.  Since pretty much everything in QUIC is varint, it sees best to stick with this (especially since it allows for a 1-byte encoding of what we care about: 0 and 1).

File: draft-ferrieuxhamchaoui-quic-lossbits.md [hunk 1]
```diff
@@ -422,7 +422,9 @@ Value: 0x1057 (if this document is approved)
 
 Parameter Name: loss_bits
 
-Specification: ..............
+Specification: Indicates that the endpoint supports loss bits. An endpoint that
```
Comment (dtikhonov) [inline draft-ferrieuxhamchaoui-quic-lossbits.md:425]: I don't think we need to recap how the transport parameter is used here -- it is done in the _Transport Parameter_ section.  We do something simple instead:

> Specification: Indicates that the endpoint supports receiving or sending loss bits.

Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:425]: I was basing this on the DATAGRAM draft (https://tools.ietf.org/html/draft-pauly-quic-datagram-05#section-7).

> Specification:  Indicates that the connection should enable support
      for unreliable DATAGRAM frames.  An endpoint that advertises this
      transport parameter can receive datagrams frames from the other
      endpoint, **up to and including the length in bytes provided in the
      transport parameter.**

Comment (dtikhonov) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:425]: I was using the [1wd draft](https://tools.ietf.org/html/draft-huitema-quic-1wd-00#section-6) for reference:

```
   Specification: Indicates that the connection should use TimeStamped
   ACK frames
```

Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:425]: @dtikhonov Well, 1wd draft defines a zero-length transport parameter. The DATAGRAM draft defines an integer parameter and asks IANA to mention the meaning of the parameter value in the registry.  We also define an integer parameter...

Comment (dtikhonov) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:425]: OK, it seems your preference is stronger than mine -- looks good to me.

Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:425]: Maybe a little stronger, though this is certainly something that could be trivially changed by the WG with 0 consequences to the design. :)

**Pull request merged**